r/DESCRIPTION

```diff
@@ -14,11 +16,11 @@ Config/rextendr/version: 0.3.1.9001
 SystemRequirements: Cargo (rustc package manager)
 Imports: 
     geoarrow,
-    arrow,
-    nanoarrow,
-    rlang
+    rlang,
+    wk
```

Comment:
```suggestion
    arrow,
    nanoarrow,
    rlang,
    wk
```

these are required for using geoarrow with arrow-extendr right now. I'd like to find a way to not need arrow but for the time being I haven't done so.